content/php/concepts/string-functions/terms/explode/explode.md

```diff
@@ -14,34 +14,55 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `explode()` function takes a string and splits it on a given delimiter string. It returns an array of the substrings produced.
+The **`explode()`** function takes a `string` and splits it on a given delimiter `string`. It returns an `array` of the substrings produced.
```

Comment:
```suggestion
The **`explode()`** function takes a string and splits it on a given delimiter string. It returns an array of the substrings produced.
```

Thanks for bolding! The string and array though aren't referring to a code snippet or actual String/Array objects so it doesn't need to be in backticks.